test/hotspot/jtreg/applications/ctw/modules/generate.bash

```diff
@@ -30,7 +30,7 @@ do
     echo creating $file for $module...
     cat > $file <<EOF
 /*
- * Copyright (c) 2017, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2017, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hm. So if the date had changed here, why it did not change in the existing files?

Reply:
1. I generated only the files for the new modules, not changing the existing ones (as they would not change);
2. I've manually edited the generated headers, removing the "2017" part. I'm not a layer, but it seems reasonable as the new files' code didn't exist before 2024, despite it being a standard and template one.

Reply:
IIRC, there is already a precedent for regenerating and updating all the files when template file change, e.g. in ByteBuffers / VarHandles. The problem with manual intervention is that a next person that regenerates the files from the template would expect no changes in files when template did not change.

Reply:
Ok, let me suggest a compromise. I modified the generator to have 2 modes - to generate a new file and to update an existing one.
Other files have been edited manually anyways (some are split in 2 parts, some have got manually added `@requires` blocks, etc.), which means that there will be no clear generation. But at least the generator became more useful and concise.